docs/inputs.md

```diff
@@ -88,7 +88,11 @@ The table below contains brief descriptions of the input files required to execu
 | [externalExternalTripsByYear.csv](#external_trip) <i> (raw inputs have these by year) <i> | External origin-destination station trip matrix | CSV | Transportation Modelers | |  |  |  | 
 | [externalInternalControlTotalsByYear.csv](#external_internal) <i> (raw inputs have these by year) <i> | External Internal station control totals read by GISDK | CSV | Transportation Modelers | |  |  |  | 
 | [internalExternal_tourTOD.csv](#internal_external_tod) | Internal-External Model tour time-of-day frequency distribution | CSV | Transportation Modelers | 
+| **CVM Establishment Synthesis** |
+| land_use | MGRA based land use file | CSV | |
+| percent_of_establishments_by_luz_size_emp_cat.xlsx | Percent of establishments in LUZ that belong in each size category by industry sector | Excel Workbook | |
 | **Commercial Vehicle Model** (TO BE UPDATED) |  |  |  | 
+| SynthEstablishments.csv | Output from CVM establishment synthesis, similar description as previous part | CSV | |
 | tazcentroids_cvm.csv | Zone centroid coordinates in state plane feet and albers | CSV | Transportation Modelers | 
```

Comment:
Please check the inputs for Commercial Vehicle Model section, and remove out dated files.